app/assets/javascripts/_selection-buttons.js

```diff
@@ -0,0 +1,16 @@
+(function(GOVUK, GDM) {
+
+  GDM.selectionButtons = function() {
```

Comment:
It's on mine and Tom's backlogs to think about how we do dependencies/modules for all the apps. TL;DR you're right, but we haven't decided what to do about it yet.